app/controllers/admin/web_hooks_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Admin::WebHooksController < Admin::AdminController
+
+  def index
+    data = {
+      web_hook_event_types: WebHookEventType.all,
+      web_hooks: WebHook.all # TODO: Pagination?
+    }
+    render_serialized(OpenStruct.new(data), AdminWebHooksSerializer, rest_serializer: true)
```

Comment:
hmm perhaps just pass in the `WebHook.all` here and add association for `WebHookEventType` in the serializer?